test/pipeline_test.py

```diff
@@ -150,6 +151,27 @@ def test_write_file():
         new_df = df + write_file('df.abc')
 
 
+def test_pivot_table():
+    pandas_df = pd.DataFrame({"A": ["foo", "foo", "foo", "foo", "foo",
+                                    "bar", "bar", "bar", "bar"],
+                              "B": ["one", "one", "one", "two", "two",
+                                    "one", "one", "two", "two"],
+                              "C": ["small", "large", "large", "small",
+                                    "small", "large", "small", "small",
+                                    "large"],
+                              "D": [1, 2, 2, 3, 3, 4, 5, 6, 7],
+                              "E": [2, 4, 5, 5, 6, 6, 8, 9, 9]})
+
+    df = DplyFrame(pandas_df)
+
+    output = df + pivot_table(values='D',
+                              index=['A', 'B'], columns=['C'], aggfunc=np.sum, fill_value=0)
+    expected = pandas_df.pivot_table(values='D', index=['A', 'B'], columns=['C'], aggfunc=np.sum, fill_value=0)
```

Comment:
Test needs to include the drop_na parameter at some point. I'll approve after that's added

Reply:
Good point! I've added it in the latest commit.